pkg/api/v1beta3/types.go

```diff
@@ -971,6 +971,16 @@ type NodeSystemInfo struct {
 	SystemUUID string `json:"systemUUID"`
 	// BootID is the boot-id reported by the node
 	BootID string `json:"bootID" description:"boot id is the boot-id reported by the node"`
+	// Kernel version reported by the node
+	KernelVersion string `json:"kernelVersion" description:"Kernel version reported by the node"`
+	// OS image used reported by the node
+	OsImage string `json:"osImage" description:"OS image used reported by the node"`
```

Comment:
What do you mean by "userland"? Like `container-vm-v20150112`? That shouldn't be here unless it's observable from the node.